Jenkinsfile

```diff
@@ -123,7 +123,8 @@ jobWrapper {
             checkLinuxDebugNoEncryp : doCheckInDocker(linuxOptionsNoEncrypt << [buildType : "Debug"]),
             checkMacOsRelease_Sync  : doBuildMacOs(buildOptions << [buildType : "Release"]),
             checkWin32Release       : doBuildWindows('Release', false, 'Win32', true),
-            checkWin32DebugUWP      : doBuildWindows('Debug', true, 'Win32', true),
+            checkWin32DebugUWP      : doBuildWindows('Debug', true, 'Win32', false),
+            checkARMDebugUWP        : doBuildWindows('Debug', true, 'ARM', false),
```

Comment:
Slight preference for `checkWinARMDebugUWP` so it is obviously windows, even if you don't know what UWP is. Not a big deal either way though.